browser/ui/webui/brave_wallet/panel_handler/wallet_panel_handler.cc

```diff
@@ -8,17 +8,21 @@
 #include <utility>
 
 #include "base/callback.h"
-#include "brave/components/permissions/contexts/brave_ethereum_permission_context.h"
+#include "brave/components/permissions/contexts/brave_wallet_permission_context.h"
 
+// It's safe bind the active webcontents when panel is created because the panel
```

Comment:
*safe to bind

Reply:
fixed in 46a8acf771da14616feee025e6f78c9cffced934